pkg/helm/build.go

```diff
@@ -90,21 +91,32 @@ func (m *Mixin) Build() error {
 	// Define kubectl
 	fmt.Fprintf(m.Out, getKubectl, kubeVersion)
 
-	// Go through repositories
-	for name, repo := range input.Config.Repositories {
-
-		commandValue, err := GetAddRepositoryCommand(name, repo.URL)
-		if err != nil && m.Debug {
-			fmt.Fprintf(m.Err, "DEBUG: addition of repository failed: %s\n", err.Error())
-		} else {
-			fmt.Fprintf(m.Out, strings.Join(commandValue, " "))
+	// Go through repositories if defined
+	if len(input.Config.Repositories) > 0 {
+		names := make([]string, 0, len(input.Config.Repositories))
+		for name := range input.Config.Repositories {
+			names = append(names, name)
+		}
+		sort.Strings(names) //sort by key we need it for the tests
```

Comment:
While I don't think sorting the repos by name is bad/wrong per se, I wonder if adding extra logic here just to satisfy testing is maybe unwarranted.  The tests can always adjust their assertions such that any order is allowed (for example, via a 'Contains' as opposed to a direct multiline match).  Not a dealbreaker by any means, but just wanted to suggest we revisit and see if this approach is worth the added code/complexity here.

In any case, if we keep the sorting, I'd suggest removing the comment about tests and updating it simply to state that we desire repeatable ordering for the repo add lines.

Reply:
You are right, it is not a good idea to add code to just satisfy testing. So the sorting was removed. Thank you